packages/mongodb/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@b-mo/mongodb",
+  "version": "0.3.0",
+  "description": "provides an interface to mongo databases",
+  "main": "index.js",
+  "author": "breul",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "mongodb": "3.3.5"
```

Comment:
this is a pretty out dated version.  If this is the continued version, we should note in the readme it's using an older version to support AWS DocDB

Reply:
Good call, I think it might be a good thing to update it so I can do that.